ui/src/app/views/home-page/component.html

```diff
@@ -0,0 +1,33 @@
+<h2>AoU Research Portal</h2>
+<div *ngIf="user else loggedout">
+  <div class="area-block">
+    <div class="table-header">
+      <!-- TODO: Add workspace link and functionality -->
+      <h3 class="table-header-descriptor">Workspaces</h3>
+      <button class="btn btn-link" (click)="addWorkspace()">Add a Workspace</button>
+    </div>
+    <div class="indented">
+      <clr-datagrid>
+        <clr-dg-column>
+          Name
+          <clr-dg-string-filter [clrDgStringFilter]="workspaceNameFilter"></clr-dg-string-filter>
+        </clr-dg-column>
+        <clr-dg-column>URL</clr-dg-column>
+        <clr-dg-column>Last Modified</clr-dg-column>
+        <clr-dg-column>
+          Research Purpose
+          <clr-dg-string-filter [clrDgStringFilter]="workspaceResearchPurposeFilter"></clr-dg-string-filter>
+        </clr-dg-column>
+        <clr-dg-placeholder>No workspaces found.</clr-dg-placeholder>
+        <clr-dg-row *clrDgItems="let workspace of workspaceList" [clrDgItem]="workspace" class="table-row workspace-table-row">
+          <clr-dg-cell><a (click)="goToWorkspace(workspace.namespace, workspace.id)">{{workspace.name}}</a></clr-dg-cell>
```

Comment:
This link doesn't look clickable until you hover over it. Can we fix that? Also, can we make this an actual link so that Command-click opens in a new tab, etc.?